csunplugged/topics/content/en/binary-numbers/unit-plan/unit-plan.yaml

```diff
@@ -5,15 +5,74 @@ how-binary-digits-work:
   duration: 45
   programming-exercises:
```

Comment:
I remember being told that no programming exercises will be linked for the 5 to 7 age bracket, so should check this at some point.